unit_tests/sources/declarative/test_concurrent_declarative_source.py

```diff
@@ -3885,7 +3885,12 @@ def test_read_concurrent_declarative_source(
         output_data = [
             message.record.data for message in _run_read(manifest, _stream_name) if message.record
         ]
-        assert output_data == expected_records
+        assert sorted(
+            output_data, key=lambda x: (x.get("partition", 0), x.get("ABC", 0), x.get("AED", 0))
+        ) == sorted(
+            expected_records,
+            key=lambda x: (x.get("partition", 0), x.get("ABC", 0), x.get("AED", 0)),
+        )
```

Comment:
i think we're also missing extracting and checking the key for `USD` so we need one more `x.get("USD", 0)`

Reply:
Thanks, @brianjlai - on it!

Reply:
Resolved!